docs-rst/installation/server-config/security.rst

```diff
@@ -29,27 +30,46 @@ You can use KSQL with a Kafka cluster in |ccloud|. For more information, see :re
 Configuring KSQL for Secured Confluent Schema Registry
 ------------------------------------------------------
 
-The following configuration connects KSQL with the Confluent Schema Registry over HTTPS.
+KSQL can be configured to connect to the Schema Registry over HTTP by setting the
+``ksql.schema.registry.url`` to the Schema Registry's HTTPS endpoint.
+Depending on your security setup, you may also need to supply a trust-store, (e.g. if the Schema
+Registry's SSL certificates are not trusted they the JVM by default), and a key-store, (e.g. if the
+Schema Registry requires mutual authentication).
 
-#. Specify the HTTPS endpoint in the ``ksql.schema.registry.url`` setting in the
-   KSQL server configuration file:
+The following configuration is all that is needed to configure KSQL to communicate with the Schema Registry
+over HTTPS, where mutual authentication is not required and the Schema Registry's SSL certificates
+are trusted by the JVM:
 
-   .. code:: bash
+.. code:: bash
+
+    ksql.schema.registry.url=https://<host-name-of-schema-registry>:<ssl-port>
+
+The following configuration can be used to configure KSQL to communicate with the Schema Registry over HTTPS,
+with mutual auth, when the SSL configuration is consistent with that needed to communicate with Kafka,
+i.e. where the configuration can be _shared_:
 
-        ksql.schema.registry.url=https://<host-name-of-schema-registry>:<ssl-port>
+.. code:: bash
 
-#. Specify any SSL or SASL configuration the Schema Registry client requires using the ``KSQL_OPTS``
-   environment variable.
+    ksql.schema.registry.url=https://<host-name-of-schema-registry>:<ssl-port>
+    ssl.truststore.location=/etc/kafka/secrets/ksql.truststore.jks
+    ssl.truststore.password=confluent
+    ssl.keystore.location=/etc/kafka/secrets/ksql.keystore.jks
+    ssl.keystore.password=confluent
+    ssl.key.password=confluent
 
-   For example, if the Schema Registry's SSL certificate is not signed by a CA that is recognized by
-   the JVM by default, then you can provide a suitable truststore when starting KSQL via the command line:
+If required, specific configuration for the Schema Registry can be supplied. The following configuration
+can be used to configure KSQL to communicate with the Schema Registry over HTTP, _without_ mutual auth,
```

Comment:
Simplify (and remove unnecessary formatting): `Use following to configure KSQL to communicate with the Schema Registry over HTTP without mutual authentication (i.e. SSL configuration is not shared):`

Reply:
As above, this reworking doesn't work. I've changed things around in my next commit. So see what you think.